engine/src/main/resources/assets/shaders/chunk_frag.glsl

```diff
@@ -277,7 +274,7 @@ void main() {
     // Apply the final lighting mix
     color.xyz *= combinedLightValue * occlusionValue;
 #else
-    gl_FragData[2].rgba = vec4(blocklightColorValue.r, blocklightColorValue.g, blocklightColorValue.b, 0.0);
+    gl_FragData[2].rgba = vec4(blocklightColorBrightness, daylightValue, 0.0, 0.0);
```

Comment:
Store both the brightness (which can be later converted again into a color value) and the daylight value.  This way we can use the #2 frag texture for more things other than just the blocklight color.